docs/source/transaction_author_agreement.md

```diff
@@ -52,29 +64,31 @@
 
 - This metadata must be signed by the user (using a standard signature used in write requests).
 
-- If Transaction Author Agreement is not set or disabled, then requests must not have any Transaction Author Agreement Acceptance metadata. 
+- If Transaction Author Agreement is not set or disabled, then requests may have any Transaction Author Agreement Acceptance metadata.
 
 - Requests to POOL and CONFIG ledgers must not have any Transaction Author Agreement Acceptance metadata.
 
 - See [requests.md](https://github.com/hyperledger/indy-node/blob/master/docs/source/requests.md) for details.
 
 #### How ledger validates Transaction Author Agreement metadata in a request
 
-- If Transaction Author Agreement is not enabled, or if this is a request to POOL or CONFIG ledger, then
+- If this is a request to POOL or CONFIG ledger, then
     
     - If the request contains Transaction Author Agreement Acceptance metadata, then REJECT it.
     
     - Otherwise - ACCEPT
 
+- If Transaction Author Agreement is not enabled, then ACCEPT
+
 - If Transaction Author Agreement is required, and the request doesn't have Transaction Author Agreement Acceptance metadata, then REJECT
 
-- If Acceptance metadata's Transaction Author Agreement digest doesn't equal to the  digest of the latest Transaction Author Agreement on the ledger - REJECT
+- If Acceptance metadata's Transaction Author Agreement digest doesn't equal to the  digest of any active Transaction Author Agreement on the ledger - REJECT
 
 - If Acceptance metadata's acceptance mechanism is not in the latest Transaction Author Agreement Acceptance Mechanisms List, then REJECT
 
 - If Acceptance metadata's acceptance time is not in the acceptable interval, then REJECT
   
```

Comment:
Please add an item about retirement check (retirement time VS PrePrepare time). Please mention that precise (not rounded time) is used for this validation